postgres/osfamilymap.yaml

```
@@ -21,10 +21,17 @@ Debian:
     {% endif %}
 
 FreeBSD:
-  conf_dir: /usr/local/pgsql/data
-  data_dir: /usr/local/pgsql/data
+    {% if repo.version >= 9.6 %}
```

This will not be evaluated correctly, since the `version` value is set as a string in the defaults and the pillar example as well. I suggest to use `float()` filter.
```suggestion
    {% if repo.version|float >= 9.6 %}
```

---

This is done. thank you.